saos-common/src/main/java/pl/edu/icm/saos/common/chart/formatter/PointWeekValueFormatter.java

```diff
@@ -0,0 +1,50 @@
+package pl.edu.icm.saos.common.chart.formatter;
+
+import org.springframework.beans.factory.annotation.Value;
+
+import pl.edu.icm.saos.common.chart.value.Week;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * 
+ * {@link PointValueFormatter} implementation handling {@link Week} class
+ * 
+ * @author Łukasz Dumiszewski
+ */
+public class PointWeekValueFormatter extends AbstractPointValueFormatter {
```

Comment:
First, I wanted to use it on ui, then I realized I needed another one (with br). I left it for the sake of rest api.